tools/genomespace/genomespace_importer.py

```diff
@@ -192,7 +191,7 @@ def download_from_genomespace_importer( username, token, json_parameter_file, ge
         # if using tmp file, move the file to the new file path dir to get scooped up later
         if using_temp_file:
             original_filename = filename
-            filename = ''.join( c in VALID_CHARS and c or '-' for c in filename )
+            filename = ''.join( c in FILENAME_VALID_CHARS and c or '-' for c in filename )
```

Comment:
Here or in a new PR I'd be very happy to see any unicode alpha-numeric character added. I'm a little more +/- 0 on white listing shell relevant characters.

When I was investigating shell-safe characters for CWL stuff - I came across this library (https://pypi.python.org/pypi/regex) - which unicode-friendly extended character classes.